libs/build.gradle

```diff
@@ -8,7 +8,7 @@ afterEvaluate {
             myLocalPublication(MavenPublication) {
                 artifact(file('lib.aar'))
                 groupId 'io.anyproto'
-                artifactId 'anytype-heart-android'
+                artifactId 'anytype-heart-android2'
```

Comment:
why changing artifact id here? to distinguish between local library and published mw library?